### PR TITLE
Update dumpling --row description

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update dumpling --row description. Add a note.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/7958
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -347,7 +347,7 @@ Finally, all the exported data can be imported back to TiDB using [TiDB Lightnin
...
-| `-r` or `--rows`             | Divide the table into specified rows of data (generally applicable for concurrent operations of splitting a large table into multiple files.                                                                                                                                                                                       |
+| `-r` or `--rows`             | Split the table into a specified row of data (generally applicable for concurrent operations of splitting a large table into multiple files. When the upstream database is TiDB v3.0 or later versions, a value of this parameter greater than 0 indicates that the TiDB region information is used for splitting and the value specified here will no longer take effect.                                                                                                                                                                                 |
```
Comment (lance6716) [inline dumpling-overview.md:350]: "a specified row of data" is a littler unclear to me, how about "pieces of data with specified number of rows"

**Review by lance6716 — COMMENTED** [review]
rest lgtm

File: dumpling-overview.md (hunk 1 above)
Comment (hfxsd) [inline dumpling-overview.md:350]:
```suggestion
| `-r` or `--rows`             | Split the table into pieces of data with specified number of rows (generally applicable for concurrent operations of splitting a large table into multiple files. When the upstream database is TiDB v3.0 or later versions, a value of this parameter greater than 0 indicates that the TiDB region information is used for splitting and the value specified here will no longer take effect.                                                                                                                                                                                 |
```

File: dumpling-overview.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ In the command above:
...
 + The `-o` option specifies the export directory of the storage, which supports a local file path or a [URL of an external storage](/br/backup-and-restore-storages.md).
 + The `-t` option specifies the number of threads for the export. Increasing the number of threads improves the concurrency of Dumpling and the export speed, and also increases the database's memory consumption. Therefore, it is not recommended to set the number too large. Usually, it's less than 64.
-+ The `-r` option specifies the maximum number of rows in a single file. With this option specified, Dumpling enables the in-table concurrency to speed up the export and reduce the memory usage.
++ The `-r` option specifies the maximum number of rows in a single file. With this option specified, Dumpling enables the in-table concurrency to speed up the export and reduce the memory usage. When the upstream database is TiDB v3.0 or later versions, a value of this parameter greater than 0 indicates that the TiDB region information is used for splitting and the value specified here will no longer take effect.
```
Comment (shichun-0415) [inline dumpling-overview.md:75]: TiDB region information is used for splitting----splitting 的对象是什么？原文是划分，好像跟 split 的意思还是有差别。

Comment (hfxsd) [reply dumpling-overview.md:75]: split的对象是table，就是把table拆分成很多行，然后并行导出。以前split和devide混用。统一改成split了。

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**